src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import racingcar.Main;
+import racingcar.util.Input;
+
+public class Cars {
+	private Map<Integer, Car> cars;
```

Comment:
```suggestion
	private List<Car> cars;
```
굳이 Map이 아니라 List로도 충분히 구현 가능할 것으로 보여짐
List로도 도전해 보면 어떨까?